assets/src/edit-story/components/panels/stylePreset/index.js

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import { rgba } from 'polished';
+import { useCallback, useEffect, useState } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as Add } from '../../../icons/add_page.svg';
+import { ReactComponent as Edit } from '../../../icons/edit_pencil.svg';
+import { ReactComponent as Remove } from '../../../icons/remove.svg';
+import { useStory } from '../../../app/story';
+import generatePatternStyles from '../../../utils/generatePatternStyles';
+import { getDefinitionForType } from '../../../elements';
+import { Panel, PanelTitle, PanelContent } from './../panel';
+
+const COLOR_HEIGHT = 35;
+
+const buttonCSS = css`
+  background: transparent;
+  width: 30px;
+  height: 28px;
+  border: none;
+  color: ${({ theme }) => rgba(theme.colors.fg.v1, 0.84)};
+  cursor: pointer;
+  padding: 0;
+`;
+
+// Since the whole wrapper title is already a button, can't use button directly here.
+const AddColorPresetButton = styled.div.attrs({
+  role: 'button',
+})`
+  ${buttonCSS}
+  svg {
+    width: 26px;
+    height: 28px;
+  }
+`;
+
+const EditModeButton = styled.div.attrs({
+  role: 'button',
+})`
+  ${buttonCSS}
+  height: 20px;
+  svg {
+    width: 16px;
+    height: 20px;
+  }
+`;
+
+const ExitEditMode = styled.a`
+  ${buttonCSS}
+  color: ${({ theme }) => theme.colors.fg.v1};
+  font-size: 12px;
+  line-height: 14px;
+  padding: 7px;
+  height: initial;
+`;
+
+const colorCSS = css`
+  display: inline-block;
+  width: 30px;
+  height: 30px;
+  border-radius: 15px;
+  border: 0.5px solid ${({ theme }) => rgba(theme.colors.fg.v1, 0.3)};
+  padding: 0;
+  svg {
+    width: 18px;
+    height: 28px;
+  }
+`;
+
+const Color = styled.button`
+  ${colorCSS}
+  background: ${({ backgroundColor, backgroundImage }) =>
+    backgroundColor ? backgroundColor : backgroundImage};
+`;
+
+// For max-height: Display 5 extra pixels to show there are more colors.
+const Colors = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  max-height: ${6 * COLOR_HEIGHT + 5}px;
+  overflow-y: scroll;
+`;
+
+const ButtonWrapper = styled.div`
+  flex-basis: 16%;
+  height: ${COLOR_HEIGHT}px;
+`;
+
+const ColorGroupLabel = styled.div`
```

Comment:
The style for this label doesn't seem super-readable. Is it like this on Figma?

Reply:
I think so, based it on Figma:
<img width="288" alt="Screenshot 2020-04-01 at 18 33 52" src="https://user-images.githubusercontent.com/3294597/78162583-5e312300-7447-11ea-97e0-82c0aae01e24.png">